test/projection_test.py

```diff
@@ -26,17 +28,17 @@ def build_constant_scan(
         raise BuildScanException
 
     scan = LaserScan()
-    scan.header.stamp = rospy.rostime.Time.from_sec(10.10)
+    scan.header.stamp = rclpy.time.Time(seconds=10.10).to_msg()
     scan.header.frame_id = "laser_frame"
     scan.angle_min = angle_min
     scan.angle_max = angle_max
     scan.angle_increment = angle_increment
-    scan.scan_time = scan_time.to_sec()
+    scan.scan_time = scan_time.nanoseconds / 1e9
     scan.range_min = PROJECTION_TEST_RANGE_MIN
     scan.range_max = PROJECTION_TEST_RANGE_MAX
     scan.ranges = [range_val for _ in range(count)]
     scan.intensities = [intensity_val for _ in range(count)]
-    scan.time_increment = scan_time.to_sec()/count
+    scan.time_increment = scan_time.nanoseconds / 1e9 / count
```

Comment:
Manually dividing by `1e9` seems a bit hacky, but I don't think that ros2 Durations have any `to_sec()` equivalent.